docs/installation.rst

```diff
@@ -306,6 +322,14 @@ To stop the containers::
 
     docker-compose stop
 
+If you have made changes to the .env or /etc/hosts file, it's a good idea to run::
+
+    docker-compose down
+    docker-compose pull
+    git pull --ff-only origin master
+    docker-compose up -d
```

Comment:
The `docker-compose pull` and `git pull --ff-only origin master` commands shouldn't be part of this, since they're not necessary after making changes to the `.env` and/or `/etc/hosts` files. I would suggest just this:
```
docker-compose down
docker-compose up -d
```
or `docker-compose stop` could be used instead of `docker-compose down`.

Reply:
I would strongly prefer `docker-compose stop` since otherwise you're losing your mounted databases and you have to start over. There are times you want to do that, but it's not related to making a change to the `.env` file. 

Also, I'm still struggling to see the value in `-d` since it can easily hide important error messages and warnings on the running processes.

Reply:
Just wanted to clear-up something here. Your mounted volumes storing your database data are **not** lost when using `docker-compose down`. You'd have to use `docker-compose down --volumes` to do that. It really doesn't matter if you use `docker-compose down` or `docker-compose stop` after updating your `.env` file. I always use `docker-compose down` out of habit (it also covers the case of running new images after doing a `docker-compose pull`), but in this case of updating the `.env` file, `docker-compose stop` does just what's needed and nothing else (like removing networks), so we can go with that.

Reply:
ahh. thanks for shedding light on that @escattone! ok, nevermind my comment then. :)

Reply:
Thank you for the clarity! 

I also removed the `-d` per Peter's comment.